README.md

```diff
@@ -27,6 +30,37 @@ gitmoji-changelog
 
 **// TODO**
 
+## Setup
+
+Install the lib in your PATH : 
+
+```bash
+npm install -g gitmoji-changelog
+```
+
+## Mapping
+
+When a commit is done with a gitmoji, the following mapping is used to group them : 
+
+| Categorie | Gitmoji |
+|---|---|
+| Added | :sparkles: :tada: :white_check_mark: :bookmark: :construction_worker: :chart_with_upwards_trend: :heavy_plus_sign: :loud_sound: |
+| Changed | :art: :zap: :lipstick: :rotating_light: :arrow_down: :arrow_up: :pushpin: :recycle: :wrench: :rewind: :alien: :truck: :boom: :bento: :wheelchair: :speech_balloon: :card_file_box: :children_crossing: :building_construction: :iphone: |
+| Removed | :fire: :heavy_minus_sign: :mute: |
+| Fixed | :bug: :ambulance: :apple: :penguin: :checkered_flag: :robot: :green_apple: :green_heart: :pencil2: |
+| Security | :lock: |
+
+
+> See [mapping.js](packages/gitmoji-changelog-core/src/mapping.js) for more details
+
+
+## Example
+
+Here is an example of the produced `changelog.md` produced : 
+
+![Changelog example](doc/screenshot.png)
```

Comment:
Sadly, we have a commit without gitmoji on the screenshot 😂

Reply:
Updated, with a border and a drop shadow :)